utl/vw-important-features

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+# ==========================================================================================
+# Find important features
+# ==========================================================================================
+
+'''
+Usage example:
+>python vw-important-features -d data.json --ml_args "--cb_adf -l 0.01 --cb_type mtr"
+
+Sample output:
+=====================================
+Testing a range of L1 regularization
+L1: 1e-06 - Num of Features: 4197
+L1: 1e-05 - Num of Features: 2741
+L1: 1e-04 - Num of Features: 46
+L1: 1e-03 - Num of Features: 13
+=====================================
+
+Inverting hashes of important features (l1 = 1e-03)
+=====================================
+Emotion0^contempt:194265:-0.0236295
```

Comment:
Wouldn't it make more sense to sort the output descending by `abs(weight)` ?

Reply:
That's a good suggestion. Right now I'm not even parsing the values. But it can be done in the future.